pygeoapi/util.py

```diff
@@ -411,6 +413,7 @@ class JobStatus(Enum):
     running = 'running'
     successful = 'successful'
     failed = 'failed'
+    dismissed = 'dismissed'
```

Comment:
Is this in spec?

Reply:
Looks like it: https://github.com/opengeospatial/ogcapi-processes/blob/954213317742fc0ae769f8c204b3712a047c8099/core/openapi/schemas/statusInfo.yaml#L15